docs/dev/run_the_etl.rst

```diff
@@ -11,24 +11,182 @@ just want to use already processed data.
 
 These instructions assume you have already gone through the :ref:`dev_setup`.
 
-There are two main scripts involved in the PUDL processing pipeline:
-
-1. ``ferc_to_sqlite`` :doc:`converts the FERC Form 1 DBF/XBRL files <clone_ferc1>` into
-   a single large `SQLite <https://sqlite.org>`__ database so that the data is easier to
-   extract, and so all of the raw FERC Form 1 data is available in a modern format. You
-   must run this script before you can run ``pudl_etl``.
-2. ``pudl_etl`` coordinates the "Extract, Transform, Load" process that processes
+Dagster
+-------
+PUDL uses `Dagster <https://dagster.io/>`__ to orchestrate data pipelines. Dagster
+makes it easy to manage data dependences, parallelize processes, cache results
+and handle IO. If you are planning on contributing to PUDL, it is recommended you
+read through the `Dagster Docs <https://docs.dagster.io/getting-started>`__ to
+familiarize yourself with the tool's main concepts.
+
+There are a handful of Dagster concepts worth understanding prior
+to interacting with the PUDL data processing pipeline:
+
+**Dagit:**
+`Dagit <https://docs.dagster.io/concepts/dagit/dagit>`__ is the Dagster
+UI for monitoring and executing ETL runs.
+
+**Software Defined Assets (SDAs):**
```

Comment:
I might change this to "Assets & Software Defined Assets (SDA)" because there were some references to assets in the rest of the docs, and initially erroneously assumed that it was talking about SDA until I went back and read it and saw that it was different.

Reply:
I didn't add "Assets" to the header so folks don't think SDAs and Assets are different but I did clarify asset is another name for SDAs.